<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoscoperMainWindow</class>
 <widget class="QMainWindow" name="AutoscoperMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1465</width>
    <height>981</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XROMM Autoscoper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_buttonAutoscoperTop">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="3">
        <widget class="QToolButton" name="toolButtonLoadTracking">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Load
Tracking</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/disk_upload.png</normaloff>:/images/resource-files/icons/disk_upload.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="toolButtonSaveTracking">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Save
Tracking</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/disk_download.png</normaloff>:/images/resource-files/icons/disk_download.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QToolButton" name="toolButtonRotate">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Rotate</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/reload.png</normaloff>:/images/resource-files/icons/reload.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="Line" name="line_2">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QToolButton" name="toolButtonTranslate">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Translate</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/arrow_nsew.png</normaloff>:/images/resource-files/icons/arrow_nsew.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QToolButton" name="toolButtonMovePivot">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Move
Pivot</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/star_gold.png</normaloff>:/images/resource-files/icons/star_gold.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <widget class="QToolButton" name="toolButtonRetrack">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Retrack</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/forward_blue.png</normaloff>:/images/resource-files/icons/forward_blue.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QToolButton" name="toolButtonTrack">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Track</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/forward_green.png</normaloff>:/images/resource-files/icons/forward_green.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="toolButtonOpenTrial">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Open
Trial</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/application_osx_go.png</normaloff>:/images/resource-files/icons/application_osx_go.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QToolButton" name="toolButtonTrackCurrent">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="text">
          <string>Track Current</string>
         </property>
         <property name="icon">
          <iconset resource="../autoscoper_resource.qrc">
           <normaloff>:/images/resource-files/icons/section_collapsed.png</normaloff>:/images/resource-files/icons/section_collapsed.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="frameWindows">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1465</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Tracking"/>
    <addaction name="actionExport_Tracking"/>
    <addaction name="separator"/>
    <addaction name="actionSaveForBatch"/>
    <addaction name="actionLoad_xml_batch"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Test_Sequence"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Background"/>
   </widget>
   <widget class="QMenu" name="menuTracking">
    <property name="title">
     <string>Tracking</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionInsert_Key"/>
    <addaction name="actionLock"/>
    <addaction name="actionUnlock"/>
    <addaction name="actionBreak_Tangents"/>
    <addaction name="actionSmooth_Tangents"/>
    <addaction name="separator"/>
    <addaction name="actionAdvanced_Settings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLayoutCameraViews"/>
    <addaction name="actionShow_world_view"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced_Tool">
    <property name="title">
     <string>Extra</string>
    </property>
    <addaction name="actionExport_NCC_as_csv"/>
    <addaction name="actionExport_all_NCCs_near_this_pose"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTracking"/>
   <addaction name="menuView"/>
   <addaction name="menuAdvanced_Tool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../autoscoper_resource.qrc">
     <normaloff>:/images/resource-files/icons/application_osx_split.png</normaloff>:/images/resource-files/icons/application_osx_split.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../autoscoper_resource.qrc">
     <normaloff>:/images/resource-files/icons/application_osx_go.png</normaloff>:/images/resource-files/icons/application_osx_go.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Tracking</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save Trial as</string>
   </property>
  </action>
  <action name="actionImport_Tracking">
   <property name="icon">
    <iconset resource="../autoscoper_resource.qrc">
     <normaloff>:/images/resource-files/icons/disk_upload.png</normaloff>:/images/resource-files/icons/disk_upload.png</iconset>
   </property>
   <property name="text">
    <string>Import Tracking</string>
   </property>
  </action>
  <action name="actionExport_Tracking">
   <property name="icon">
    <iconset resource="../autoscoper_resource.qrc">
     <normaloff>:/images/resource-files/icons/disk_download.png</normaloff>:/images/resource-files/icons/disk_download.png</iconset>
   </property>
   <property name="text">
    <string>Export Tracking</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionInsert_Key">
   <property name="text">
    <string>Insert Key</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="text">
    <string>Lock</string>
   </property>
  </action>
  <action name="actionUnlock">
   <property name="text">
    <string>Unlock</string>
   </property>
  </action>
  <action name="actionBreak_Tangents">
   <property name="text">
    <string>Break Tangents</string>
   </property>
  </action>
  <action name="actionSmooth_Tangents">
   <property name="text">
    <string>Smooth Tangents</string>
   </property>
  </action>
  <action name="actionLayoutCameraViews">
   <property name="text">
    <string>Layout Camera Views</string>
   </property>
  </action>
  <action name="actionSaveForBatch">
   <property name="text">
    <string>save xml for batch</string>
   </property>
  </action>
  <action name="actionLoad_xml_batch">
   <property name="text">
    <string>load xml batch</string>
   </property>
  </action>
  <action name="actionSave_Test_Sequence">
   <property name="text">
    <string>Save Test Sequence</string>
   </property>
  </action>
  <action name="actionShow_world_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show world view</string>
   </property>
  </action>
  <action name="actionSet_Background">
   <property name="text">
    <string>Set Background</string>
   </property>
  </action>
  <action name="actionExport_NCC_as_csv">
   <property name="text">
    <string>Export NCC values</string>
   </property>
  </action>
  <action name="actionExport_all_NCCs_near_this_pose">
   <property name="text">
    <string>Export all NCCs near this pose</string>
   </property>
  </action>
  <action name="actionAdvanced_Settings">
   <property name="text">
    <string>Advanced Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../autoscoper_resource.qrc"/>
 </resources>
 <connections/>
</ui>
